data-plane/core/src/test/java/dev/knative/eventing/kafka/broker/core/metrics/MetricsTest.java

```diff
@@ -36,7 +36,8 @@
 public class MetricsTest {
 
     static {
-        BackendRegistries.setupBackend(new MicrometerMetricsOptions().setRegistryName(Metrics.METRICS_REGISTRY_NAME));
+        BackendRegistries.setupBackend(
+                new MicrometerMetricsOptions().setRegistryName(Metrics.METRICS_REGISTRY_NAME), null);
```

Comment:
hey @tsegismont :wave: 

Your API changed, based on what I saw for its `impl`, I am going w/ `null` as the second argument. (for all the occurrences)

Reply:
That should work